docs/source/config/settings.rst

```diff
@@ -34,6 +34,12 @@ Authentication
 
     Default: ``True``
 
+.. data:: LOCAL_GROUPS
+
+    This setting controls whether local Indico groups are available.
```

Comment:
I'd add this below your text:

```
If no centralized authentication infrastructure that supports groups (e.g. LDAP)
is used, local groups are the only way to define groups in Indico, but if you do
have central groups it may be useful to disable local ones to have all groups
in one central place.
```